src/content/configuration/output.md

```diff
@@ -293,13 +293,13 @@ If using the [`output.library`](#output-library) option, the library name is aut
 
 `string` or `object` (since webpack 3.1.0; for `libraryTarget: "umd"`)
 
-Use `library`, and `libraryTarget` below, when writing a JavaScript library that should export values, which can be used by other code depending on it. Pass a string with the name of the library:
+Use `output.library`, and `output.libraryTarget` below, when writing a JavaScript library that should export values, which can be used by other code depending on it. Pass a string with the name of the library:
```

Comment:
@metatoaster Here are you trying to say that you pass a string with the name of the library, and you add or replace that name if the library should export values? Or something else?

Reply:
This really could be elaborate more - I did not change much from the original text (see that the change was adding the `output.` prefix.  The context is that the explicit details as to what the values do are listed below that, so this fact should be made more clear.

Reply:
Actually, I scrapped that entire sentence and example, it contributed nothing.